src/partials/contacts.html

```diff
@@ -0,0 +1,103 @@
+<section class="contacts">
+    <div aria-labelledby="head">
+        <h2 class="contacts__head" id="head"> Ice Cream</h2>
+    </div>
+<ul class="contacts__box">
+    <li class="contacts__container">
+        <div class=" contacts__content">
```

Comment:
nie ma potrzeby oplatać zawartości li tagiem div dlatego że li robi to samo (oplata swoją zawartość)